### PR TITLE
add waiters/explicit

### PR DESCRIPTION
**Pull request by amyrchyk**

### CODE REVIEW

**Comment by vrudyk** [comment]
ok

File: src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/android/CarinaDescriptionPage.java [hunk 1]
```diff
@@ -32,40 +34,43 @@
...
-        super(driver);
+	super(driver);
     }
 
     @Override
     public WebViewPageBase navigateToWebViewPage() {
-        leftMenuButton.click();
-        webViewLink.click();
-        return initPage(getDriver(), WebViewPageBase.class);
+	new WebDriverWait(driver, 10).until(ExpectedConditions.presenceOfElementLocated(leftMenuButton.getBy()));
```
Comment (vrudyk) [inline src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/android/CarinaDescriptionPage.java:42]: you can create variable wait and use for all lines where you use 'new WebDriverWait(driver, 10)'

File: src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/android/CarinaDescriptionPage.java [hunk 2]
```diff
@@ -32,40 +34,43 @@
...
+	webViewLink.click();
+	return initPage(getDriver(), WebViewPageBase.class);
     }
 
     @Override
     public ChartsPageBase navigateToChartsPage() {
-        leftMenuButton.click();
-        chartsLink.click();
-        return initPage(getDriver(), ChartsPageBase.class);
+	new WebDriverWait(driver, 10).until(ExpectedConditions.presenceOfElementLocated(leftMenuButton.getBy()));
```
Comment (vrudyk) [inline src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/android/CarinaDescriptionPage.java:50]: see above

File: src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/android/CarinaDescriptionPage.java [hunk 3]
```diff
@@ -32,40 +34,43 @@
...
+	chartsLink.click();
+	return initPage(getDriver(), ChartsPageBase.class);
     }
 
     @Override
     public MapsPageBase navigateToMapPage() {
-        leftMenuButton.click();
-        mapLink.click();
-        return initPage(getDriver(), MapsPageBase.class);
+	new WebDriverWait(driver, 10).until(ExpectedConditions.presenceOfElementLocated(leftMenuButton.getBy()));
```
Comment (vrudyk) [inline src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/android/CarinaDescriptionPage.java:58]: see above

File: src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/android/CarinaDescriptionPage.java [hunk 4]
```diff
@@ -32,40 +34,43 @@
...
+	mapLink.click();
+	return initPage(getDriver(), MapsPageBase.class);
     }
 
     @Override
     public UIElementsPageBase navigateToUIElementsPage() {
-        leftMenuButton.click();
-        uiElementsLink.click();
-        return initPage(getDriver(), UIElementsPageBase.class);
+	new WebDriverWait(driver, 10).until(ExpectedConditions.presenceOfElementLocated(leftMenuButton.getBy()));
```
Comment (vrudyk) [inline src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/android/CarinaDescriptionPage.java:66]: see above

File: src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/android/CarinaDescriptionPage.java (hunk 1 above)
Comment (amyrchyk) [reply src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/android/CarinaDescriptionPage.java:42]: done

File: src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/android/CarinaDescriptionPage.java (hunk 2 above)
Comment (amyrchyk) [reply src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/android/CarinaDescriptionPage.java:50]: done

File: src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/android/CarinaDescriptionPage.java (hunk 3 above)
Comment (amyrchyk) [reply src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/android/CarinaDescriptionPage.java:58]: done

File: src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/android/CarinaDescriptionPage.java (hunk 4 above)
Comment (amyrchyk) [reply src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/android/CarinaDescriptionPage.java:66]: done

**Comment by vrudyk** [comment]
ok

**Pull request merged**